Sources/AccessibilitySnapshot/Core/Swift/Classes/AccessibilitySnapshotView.swift

```diff
@@ -594,6 +637,148 @@ private extension AccessibilitySnapshotView {
 
 }
 
+
+// MARK: -
+
+private final class CustomContentView: UIView {
```

Comment:
@NickEntin Not an item for this PR, but this file is starting to get fairly large. Would it be worthwhile refactoring the subclasses into their own files?

Reply:
Yeah, it's probably time 👍